### DISCUSSION TITLE
How to disable swagger endpoints for emqx

### DISCUSSION THREAD

**Discussion by bingfenglai** [emqx/emqx#12297] [General]
How to disable swagger endpoints for emqx.

**Comment by lafirest** [comment]
@bingfenglai 
No, currently, there is no way to disable it, but we could add a switch in configuration if needed

**Comment by lafirest** [comment]
@wivwiv 
please have a look at this

**Comment by ieQu1** [comment]
Can you explain why is this needed? You can disable the entire REST API listener.

**Comment by lafirest** [comment]
@bingfenglai 
exposed a switch by this #12398
